runtime-light/stdlib/rpc/rpc-api.cpp

```diff
@@ -404,7 +404,7 @@ bool is_int32_overflow(int64_t v) noexcept {
 }
 
 void store_raw_vector_double(const array<double> &vector) noexcept { // TODO: didn't we forget vector's length?
-  RpcComponentContext::get().rpc_buffer.store_bytes(reinterpret_cast<const char *>(vector.get_const_vector_pointer()), sizeof(double) * vector.count());
+  RpcComponentContext::get().rpc_buffer.store_bytes({reinterpret_cast<const char *>(vector.get_const_vector_pointer()), sizeof(double) * vector.count()});
```

Comment:
I would introduce new variable here for view, too hard to read

Reply:
Done